src/pages/Download.js

```diff
@@ -33,299 +33,397 @@ const Download = () => {
-                </TabSection>
-              </TabPanel>
-              <TabPanel>
-                <TabSection>
-                  <div>
-                    <SectionTitle>Experimental Builds</SectionTitle>
-                    <Button
-                      className={'mt-2'}
-                      href={
-                        'https://mega.nz/folder/t9EEFSaS#JPiOPSInCZyU-SW_-rhEOQ'
-                      }
-                      target={'_blank'}>
-                      Browse or download
-                    </Button>
-                  </div>
-                  <div>
-                    <p>NOTE:</p>
-                    <BulletListInside>
-                      <li>The repository is hosted inside a MEGA folder</li>
-                    </BulletListInside>
-                  </div>
-                </TabSection>
-                <br />
-                <TabSection>
-                  <div>
-                    <SectionTitle>Requirements</SectionTitle>
-                    <BulletListInside>
-                      <li>CPU: 64-bit, Dual-Core 2GHz CPU with SSE2 support</li>
-                      <li>RAM: minimum 2GB</li>
-                      <li>Display: 1280×768</li>
-                      <li>GPU: minimum 1GB VRAM; OpenGL 3.3+</li>
-                      <li>Hard Drive: minimum 500MB free space</li>
-                    </BulletListInside>
-                  </div>
-                </TabSection>
+                <Tabs forceRenderTabPanel>
+                  <TabSection>
+                    <TabList>
+                      <Tab>Windows</Tab>
+                      <Tab>Linux</Tab>
+                      <Tab>macOS</Tab>
+                      <Tab>Experimental</Tab>
+                    </TabList>
+                    <TabPanel>
+                      <TabSection>
+                        <div>
+                          <SectionTitle>Windows 8.1, 10, & 11</SectionTitle>
+                          <Button
+                            href={'https://github.com/UPBGE/upbge/releases/download/v0.2.5b/UPBGEv0.2.5b-b2.79Windows64.7z'}
+                            target={'_blank'}>Download (64-bit)
+                          </Button>
+                        </div>
+                        <div>
+                        <Button
+                          href={'https://github.com/UPBGE/upbge/releases/download/v0.2.5b/UPBGEv0.2.5b-b2.79Windows32.7z'}
+                          target={'_blank'}>Download (32-bit)
+                        </Button>
+                        </div>
+                        <div>
+                          <SectionTitle>Requirements</SectionTitle>
+                          <BulletListInside>
+                            <li>Blender 2.79 runs on all systems that support</li>
+                            <li className="list-none ml-4">OpenGL 2.1 and above, with recent</li>
+                            <li className="list-none ml-4">graphics drivers. For macOS, version 10.9 and later are supported.</li>
```

Comment:
Remove macOS requirement

Reply:
The Mac requirements for 2.7.9 are officially stated on the Blender website.
Are you sure you want to remove them?
<https://www.blender.org/download/requirements/>

Reply:
I mean removing it from 0.2.x Windows and Linux tab and leave it in macOS tab only.

Reply:
You're absolutely correct. How did I miss that!
Thank you. Will definitely fix that too.